showcase/tests/integration/components/hds/form/section/index-test.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'showcase/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/form/section/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it should render the component with a CSS class that matches the component name', async function (assert) {
+    await render(hbs`<Hds::Form::Section id="test-form-section" />`);
+    assert.dom('#test-form-section').hasClass('hds-form__section');
+  });
+
+  // CONTENT
+
+  test('it should yield the Section Header, HeaderTitle/Description, SectionMultiFieldGroup, SectionMultiFieldGroupItem, and Section children', async function (assert) {
+    await render(
+      hbs`<Hds::Form::Section id="test-form-section" as |Section|>
+            <Section.Header>
+              <Section.HeaderTitle />
+              <Section.HeaderDescription />
+            </Section.Header>
+            <Section.MultiFieldGroup as |FG|>
+              <FG.Item />
+            </Section.MultiFieldGroup>
```

Comment:
[todo] same as above, we want to test the yielding of the `section/index.hbs` file here
```suggestion
            <Section.MultiFieldGroup>
              <Section.MultiFieldGroupItem />
            </Section.MultiFieldGroup>
```